README.md

```diff
@@ -1 +1,14 @@
 [![codecov](https://codecov.io/gh/CruGlobal/kotlin-mpp-godtools-tool-parser/branch/main/graph/badge.svg)](https://codecov.io/gh/CruGlobal/kotlin-mpp-godtools-tool-parser)
+
+# Publishing
+
+## SNAPSHOT builds
+This project automatically publishes SNAPSHOTs to both a Maven and Cocoapods repo on any push to the `main` branch. 
+
+## Release builds
+Steps to publish a release build
+- Create a git tag that points at main.
+  - The name of this tag should be `v{release_version}`, with `release_version` matching the version specified in the `gradle.properties` file
+  - This will trigger a GitHub Actions workflow that will publish artifacts in the correct locations
+- Increment the version found in `gradle.properties` on the `main` branch
```

Comment:
Should this be done via pull request?

Reply:
it can be, I just do it directly on the main branch as part of the release process. basically after we push the release version we want to increment the version immediately so that any future updates have the correct version.

Reply:
Let's put one way or the other in the documentation.

Reply:
this is all automated now